OasisDEX/oasis-borrow
# Update the reward UI for YieldETH/ETH and YieldBTC/WBTC Pools

# [Update the reward UI for YieldETH/ETH and YieldBTC/WBTC Pools](https://app.shortcut.com/oazo-apps/story/11638/update-the-reward-ui-for-yieldeth-eth-and-yieldbtc-wbtc-pools)

## Changes 👷‍♀️
- Add YIELDBTC-WBTC and YIELDETH-ETH to `AJNA_POOLS_WITH_REWARDS` config.
  
## How to test 🧪

1. Patch Product Finder data.
2. Check if sparks tooltip is visible in Product Finder row with YIELD tokens.
3. Check if rewards banner is visible in YIELD token pools on open and manage flow pages.
